src/components/PopupWithConfirmation.js

```diff
@@ -12,7 +12,7 @@ renderLoading(isLoading, loadingText = "Deleting...") {
     if (isLoading) {
         this._submitButton.textContent = loadingText;
     } else {
-      this._submitButtonText = "Save";
+      this._submitButtonText = "Delete";
```

Comment:
`this._submitButtonText = "Delete";`  should be deleted. You have the correct `this._submitButtonText` in line 8 with text `Yes` (it's fixed in the constructor authomatically)